subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1337,6 +1331,24 @@ int ull_conn_llcp(struct ll_conn *conn, uint32_t ticks_at_expire, uint16_t lazy)
 		}
 	}
 
+#if defined(CONFIG_BT_CTLR_PERIPHERAL_ISO)
+	/* In any state, allow processing of CIS peripheral waiting for
+	 * instant.
+	 */
+	if (conn->llcp_cis.state == LLCP_CIS_STATE_INST_WAIT) {
+		struct lll_conn *lll = &conn->lll;
+		uint16_t event_counter;
+
+		/* Calculate current event counter */
+		event_counter = lll->event_counter +
+				lll->latency_prepare + lazy;
```

Comment:
It seems like the `event_counter` can be calculated using a function taking a `const ll_conn *conn` and a `uint16_t lazy` as we are doing the same calculation multiple places

Reply:
Was thinking the same thing, but as this is legacy LLCP code, with the new one coming, I didn't bother.